docs/libauthor_docs.rst

```diff
@@ -0,0 +1,60 @@
+.. currentmodule:: uarray
+
+.. _libauthordocs:
+
+Documentation for backend providers
+===================================
+
+Backend providers need to be aware of three protocols: ``__ua_domain__``,
+``__ua_function__`` and ``__ua_convert__``. These three must be a part of
+every backend object.
```

Comment:
the docs are clear, it just raises the question why? I imagine that coercion would not make sense for many objects.

Reply:
Ah, yes... So the reverse dispatcher and the coercion wouldn't even be necessary. 😄